src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -140,30 +152,37 @@ public static class ClientOperations
         // and the delegate to invoke for it, provided with the HttpClient instance on which to make the call and
         // returning asynchronously the retrieved response string from the server.  Individual operations can be
         // commented out from here to turn them off, or additional ones can be added.
-        public static (string, Func<RequestContext, Task>)[] Operations =>
+        public static (string name, Func<RequestContext, Task> operation)[] Operations =>
             new (string, Func<RequestContext, Task>)[] 
             {
                 ("GET",
                 async ctx =>
                 {
-                    Version httpVersion = ctx.GetRandomHttpVersion();
-                    using (var req = new HttpRequestMessage(HttpMethod.Get, "/") { Version = httpVersion })
+                    using (var req = new HttpRequestMessage(HttpMethod.Get, "/"))
                     using (HttpResponseMessage m = await ctx.SendAsync(req))
                     {
-                        ValidateHttpVersion(m, httpVersion);
+                        ValidateHttpVersion(m, ctx.HttpVersion);
                         ValidateStatusCode(m);
-                        ValidateContent(ctx.ContentSource, await m.Content.ReadAsStringAsync());
+                        string content = await m.Content.ReadAsStringAsync();
```

Comment:
Why can't ValidateContent be used here anymore?

Reply:
It's not comparing against a string that is known to the client.